Phonebook.Frontend/src/i18n/messages.de.xlf

```diff
@@ -1262,6 +1311,23 @@ Diese Seite nutzt Cookies, um die Bedienung der Seite zu verbessern. Cookies wer
         <target state="translated">Gebäude</target>
         <note priority="1" from="description">Title of Table Column "Building"</note>
         <note priority="1" from="meaning">TableComponent</note>
+      </trans-unit><trans-unit id="User-InformationDialogSubject" datatype="html">
+        <source>There is an Issue with your Phonebook Profil</source><target state="translated">Fehler auf deinem Phonebookprofil</target>
+        <note priority="1" from="description">Send a mail to your mate if there is something wrong on the profile </note>
+        <note priority="1" from="meaning">MailToMateSubject</note>
+      </trans-unit><trans-unit id="User-InformationDialogGreeting" datatype="html">
+        <source>Hi </source><target state="translated">Hallo </target>
+        <note priority="1" from="description">Send a mail to your mate if there is something wrong on the profile </note>
+        <note priority="1" from="meaning">MailToMateGreeting</note>
+      </trans-unit><trans-unit id="User-InformationDialogBody" datatype="html">
+        <source>, 
+ while browsing your profile I noticed that something is not right: 
+ Please contact the HR Department to fix it. This is the Phonebook Link: </source><target state="translated">, 
+ beim besuchen deines Profils, ist mir aufgefalles das es Fehler gibt: 
```

Comment:
same her. Or is this not easy to fix?
![image](https://user-images.githubusercontent.com/39745446/70147731-2ab2d780-16a5-11ea-80f0-df8ed41b146d.png)